ci/templates/kokoro/docker-fragments.sh

```diff
@@ -43,6 +43,21 @@ RUN wget -q https://github.com/google/googletest/archive/release-1.10.0.tar.gz &
     ldconfig
 _EOF_
 
+read_into_variable INSTALL_BENCHMARK_FROM_SOURCE <<'_EOF_'
```

Comment:
Can we call this `INSTALL_GOOGLE_BENCHMARK_FROM_SOURCE`?  The named `_BENCHMARK_` is a bit too generic maybe.

Reply:
Done.